src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleEqualityTests.cs

```diff
@@ -0,0 +1,2957 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.CodeGen
+{
+    [CompilerTrait(CompilerFeature.TupleEquality)]
+    public class CodeGenTupleEqualityTests : CSharpTestBase
```

Comment:
Is there a test that demonstrates we skip later element comparisons if an earlier element comparison fails?

Reply:
Updated `TestEvaluationOrderOnTupleLiteral`

---
In reply to: [170413228](https://github.com/dotnet/roslyn/pull/23356#discussion_r170413228) [](ancestors = 170413228)